ads/google/a4a/utils.js

```diff
@@ -156,6 +171,9 @@ export function googleAdUrl(
         {name: 'brdim', value: additionalDimensions(win, viewportSize)},
         {name: 'isw', value: viewportSize.width},
         {name: 'ish', value: viewportSize.height},
+        // TODO(levitzky) Look into expandable support.
+        {name: 'ea', value: '0'},
+        {name: 'pfx', value: pfx},
```

Comment:
unless you do have some other place to look up, otherwise, i think this is probably the best place